docs/content/dagster-cloud.mdx

```diff
@@ -118,15 +118,19 @@ Learn how to deploy your code to Dagster Cloud, use command line tools, and set
 
 <ArticleList>
   <ArticleListItem
-    title="Using the Dagster Cloud CLI"
+    title="Using the dagster-cloud CLI"
     href="/dagster-cloud/developing-testing/dagster-cloud-cli"
   ></ArticleListItem>
   <ArticleListItem
-    title="Adding code locations"
+    title="Adding and managing Dagster Code"
     href="/dagster-cloud/developing-testing/code-locations"
   ></ArticleListItem>
   <ArticleListItem
-    title="Using Branch Deployments"
+    title="Using Branch Deployments for Continous Integration"
```

Comment:
nit: can we add ` (CI)` at the end of the title because this might the word people would look for. [1]